packages/admin-panel/src/editor/actions.js

```diff
@@ -207,16 +207,16 @@ export const saveEdits =
       });
 
       if (filesByFieldKey && Object.keys(filesByFieldKey).length > 0) {
-        const request = isNew ? api.multipartPost : api.multipartPut;
-        await request({
+        const request = isNew ? 'multipartPost' : 'multipartPut';
+        await api[request]({
```

Comment:
TupaiaApi `this` context got lost when assigning to request, so I had to do this instead